emeis/core/migrations/0006_rename_meta_field_metainfo.py

```diff
@@ -0,0 +1,58 @@
+# Generated by Django 3.2.11 on 2022-01-28 11:10
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("emeis_core", "0005_manager_on_user_model"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="acl",
+            name="meta",
+        ),
+        migrations.RemoveField(
+            model_name="permission",
+            name="meta",
+        ),
+        migrations.RemoveField(
+            model_name="role",
+            name="meta",
+        ),
+        migrations.RemoveField(
+            model_name="scope",
+            name="meta",
+        ),
+        migrations.RemoveField(
+            model_name="user",
+            name="meta",
+        ),
+        migrations.AddField(
+            model_name="acl",
+            name="metainfo",
+            field=models.JSONField(default=dict, verbose_name="metainfo"),
+        ),
+        migrations.AddField(
+            model_name="permission",
+            name="metainfo",
+            field=models.JSONField(default=dict, verbose_name="metainfo"),
+        ),
+        migrations.AddField(
+            model_name="role",
+            name="metainfo",
+            field=models.JSONField(default=dict, verbose_name="metainfo"),
+        ),
+        migrations.AddField(
+            model_name="scope",
+            name="metainfo",
+            field=models.JSONField(default=dict, verbose_name="metainfo"),
+        ),
+        migrations.AddField(
+            model_name="user",
+            name="metainfo",
+            field=models.JSONField(default=dict, verbose_name="metainfo"),
+        ),
```

Comment:
We can't just drop data like that. Django should in theory be able to detect renames, but if it didn't, you need to change the migration explicitly: https://docs.djangoproject.com/en/4.0/ref/migration-operations/#renamefield

Reply:
it didn't detect it (and repeatedly does not). In order to avoid two migrations I have added a migration to rename and alter verbose name to maintain the existing structure